db/migrate/20201012120000_update_vms_table_comment.rb

```diff
@@ -0,0 +1,5 @@
+class UpdateCommentToVmsTableFormat < ActiveRecord::Migration[5.0]
```

Comment:
The class name must match the underscored file name (so one of them needs to adjust).  Additionally, this should be `[5.2]`

Reply:
Updated